src/value/mod.rs

```diff
@@ -57,8 +57,8 @@
 //!
 //! let owned_elem = OwnedElement::new(
 //!     vec![
-//!         local_sid_token(300).with_source("foo", 12),
-//!         text_token("hello")
+//!         owned_local_sid_token(300).with_source("foo", 12),
+//!         owned_text_token("hello")
 //!     ],
 //!     OwnedValue::String("world".into())
```

Comment:
Should the borrowed symbol token below be rewritten to use these constructor functions (IIRC `OwnedSymbolToken` doesn't have a `new` anymore, we should probably do the same to `BorrowedSymbolToken`).  Those should probably be aliased to be consistent with this.